### PR TITLE
Adding some more technologies

### PR DESCRIPTION
**Pull request by shivakrishna9**
Seems that we need to add more technologies. Pls suggest what else to add.

### CODE REVIEW

**Comment by duttakapil** [comment]
Rather than us manually entering technologies, we need to be able to let the user enter whatever technologies they are familiar with or interested in. I believe @vinaypuppal has already worked this out

**Comment by vinaypuppal** [comment]
@shivakrishna9 Thanks for your PR. 
But as @duttakapil  said users can add additional technologies like show below so closing this.
screen shot 2017-08-09 at 4 33 20 pm
screen shot 2017-08-09 at 4 33 34 pm

File: pages/profile/new.js [hunk 1]
```diff
@@ -113,7 +117,7 @@ class Profile extends React.Component {
       });
       return;
     }
-    if (!company) {
+    if (profession=='working' && !company) {
```
Comment (vinaypuppal) [inline pages/profile/new.js]: @shivakrishna9 This is bug. Can you can send this fix in separate  PR.

Comment (vinaypuppal) [inline pages/profile/new.js]: Just saw your another PR #5 Thanks.

**Pull request closed without merging**